client.go

```diff
@@ -65,9 +65,9 @@ func (c *Client) Resolve(ctx context.Context, did string) (*Doc, error) {
 	return &doc, nil
 }
 
-func (c *Client) Submit(ctx context.Context, did string, op Operation) (*LogEntry, error) {
```

Comment:
I could go either way on whether this should return (and print) the LogEntry. feels like a nice confirmation, but not really required eithr.

Reply:
The LogEntry isn't actually returned in the submission http response (just the string "OK" (https://github.com/did-method-plc/did-method-plc/blob/60e815821ddb077ce8157534341eb9abda6ef708/packages/server/src/routes.ts#L118)), so we'd have to fire off a separate request for that if we want it